packages/eslint-config-angular-ts/package.json

```diff
@@ -18,11 +18,13 @@
   },
   "peerDependencies": {
     "eslint": ">=7",
+    "eslint-plugin-jasmine": "^4.1.3",
```

Comment:
Peer dependencies should not have an upper version limit. Here we are preventing projects using this config to install for example eslint-plugin-jasmine v5. This should be changed to >=4 (look at other peer dependencies).

Let's change the dev dependency as well.